README.md

```diff
@@ -259,6 +259,38 @@ when there are no arguments on the same line as the function name.
         (foo x 1))
       ([x y]
         (+ x y)))
+* <a name="multiple-arity-order"></a> Sort the arities of a function
```

Comment:
This doesn't look right. The new bullet point seems to be within the code block of the previous bullet point.